src/back/spv/writer.rs

```diff
@@ -439,6 +441,44 @@ impl Writer {
             results: Vec::new(),
         };
 
+        for (handle, expression) in ir_function.expressions.iter() {
+            match expression {
```

Comment:
let's `match *expression` instead, like in all of the other places

Reply:
Done!